articles/azure-monitor/app/java-standalone-profiler.md

```diff
@@ -89,6 +89,22 @@ In this scenario, a profile will occur in the following circumstances:
 - Full garbage collection is executed
 - The Tenured regions occupancy is above 691 mb after collection
 
+### Request
+
+SLA triggers are based on OpenTelemetry (otel) and they will initiate a profile if certain criteria is fulfilled.
+
+Each individual trigger configuration is formed as follows:
+
+- `Name` - A unique identifier for the trigger.
+- `Filter` - Filters the requests of interest for the trigger.
+- `Aggregation` - This calculates the ratio of requests that breached a given threshold.
+  - `Threshold` - A value (in milliseconds) above which a request breach is determined.
+  - `Minimum samples` - The minimum number of samples that must be collected for the aggregation to produce data, this is to prevent triggering off of small sample sizes.
+  - `Window` - Rolling time window (in milliseconds).
+- `Threshold` - The threshold value (percentage) applied to the aggregation output. If this value is exceeded, a profile is initiated.
+
+For instance, consider a scenario where you want to trigger a profile when more than 30% of requests to a specific endpoint (/users/.*) take longer than 50 ms within a 10000 ms window.
```

Comment:
Were you intending to add an example below this?

Reply:
I added it